front/app/containers/IdeasShowPage/DesktopTopBar.tsx

```diff
@@ -53,16 +38,12 @@ const TopBar = ({ project, idea }: Props) => {
   }, [goBack, project]);
 
   return (
-    <Bar>
-      <Box mb="40px">
-        <GoBackButtonSolid
-          text={formatMessage(messages.goBack)}
-          onClick={handleGoBack}
-        />
-      </Box>
-
-      <IdeaMoreActions idea={idea} projectId={project.id} />
```

Comment:
As stated earlier, this is done so we can take the go back button out of the main landmark.